stage_descriptions/consuming-messages-02-dh6.md

```diff
@@ -22,7 +22,6 @@ The tester will validate that:
 - The correlation ID in the response header matches the correlation ID in the request header.
 - The error code in the response body is `0` (No Error).
 - The `throttle_time_ms` field in the response is `0`.
-- The `session_id` field in the response is `0`.
```

Comment:
@UdeshyaDhungana is this something we were validating before? And even if not, is there a reason we aren't validating `session_id` to be 0?